src/PostRector/Rector/ClassRenamingPostRector.php

```diff
@@ -95,6 +95,11 @@ public function shouldTraverse(array $stmts): bool
         return $this->oldToNewClasses !== [];
     }
 
+    public function getPriority(): int
```

Comment:
Add new priorities to the existing post rectors to make sure that they are sorted in the right way and will be executed in the same order as they were executed previously